package.json

```diff
@@ -9,6 +9,8 @@
         "react-redux": "^5.0.7",
         "react-scripts": "1.1.4",
         "redux": "^4.0.0",
+        "redux-thunk": "^2.2.0",
+        "rxjs": "^6.2.0",
```

Comment:
is RX needed if we have `redux-thunk`?

Reply:
I see we don't import anything from this library, so I guess you pushed it accidentally

Reply:
you are right, thanks!

Reply:
I need to investigate it further, as without it lint-stage is throwing this error:
https://github.com/okonet/lint-staged/issues/416

Reply:
fixed in https://github.com/warsawjs/topics-manager/pull/11/commits/7646cd73b9ac06daa8c4cbdec9a383bc2902d39b
Please note that in my case it was necessary to totally remove `lint-staged` from both node_modules and package.json and install it again :(